pkg/users/account-details.js

```diff
@@ -200,15 +204,16 @@ export function AccountDetails({ accounts, groups, shadow, current_user, user })
 
     if (!account) {
         return (
-            <div id="account-failure" className="curtains-ct blank-slate-pf">
-                <div className="blank-slate-pf-icon">
-                    <i className="fa fa-exclamation-circle" />
-                </div>
-                <h1>{_("Account not available or cannot be edited.")}</h1>
+            <EmptyState variant={EmptyStateVariant.small} id="account-failure">
+                <EmptyStateIcon icon={ExclamationCircleIcon} />
+                <Title headingLevel="h1" size="lg">
+                    {_("Account not available or cannot be edited.")}
+                </Title>
                 <Breadcrumb>
```

Comment:
Empty state has `<EmptyStateSecondaryActions>` where this Breadcrum would fit nicely. Or you can use or custom `EmptyStatePanel` which we use everywhere else.